zapisy/apps/common/assets/main/expose_libs.ts

```diff
@@ -1,13 +1,16 @@
 import $ from "jquery";
 import Popper from "popper.js";
+import * as bootstrap from "bootstrap";
 
 (window as any).$ = $;
 (window as any).jQuery = $;
 (window as any).Popper = Popper;
 
-import "bootstrap";
-
-$(function () {
-  // Enable Bootstrap popovers on entire site.
-  $('[data-toggle="popover"]').popover();
+window.addEventListener("load", () => {
+  const popoverTriggerList = [].slice.call(
+    document.querySelectorAll('[data-bs-toggle="popover"]')
+  );
```

Comment:
```suggestion
  const popoverTriggerList = Array.from(
    document.querySelectorAll('[data-bs-toggle="popover"]')
  );
```
Ja chyba nawet wiem, skąd wziął się zaproponowany kod, ale myślę, że to niepotrzebna "obfuskacja" – moja propozycja jest chyba czytelniejsza, a robi to samo.

Plus (ale tu już nie wiem, czy to poprawia czytelność) mam pewną niechęć do przypisywania wartości pod identyfikator jeśli jedyne, co z nim robimy, to wywołanie na nim metody w kolejnym wierszu.

Reply:
Zmienione, poprzedni kod sugerowała wprost dokumentacja bootstrapa.